package.json

```diff
@@ -8,7 +8,8 @@
     "start": "react-native start",
     "test": "jest --silent --verbose",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix": "eslint . --fix --ext .js,.jsx,.ts,.tsx"
+    "lint:fix": "eslint . --fix --ext .js,.jsx,.ts,.tsx",
```

Comment:
You can add here a ci specific linting command directly:

```
"lint:ci": "yarn run eslint --quiet --format junit -o reports/lint/junit-lint.xml",
```
Mabye it is easier to do this now.